src/libcore/ptr.rs

```diff
@@ -2903,7 +2914,9 @@ impl<T: Sized> NonNull<T> {
     /// some other means.
     #[stable(feature = "nonnull", since = "1.25.0")]
     #[inline]
-    pub fn dangling() -> Self {
+    #[cfg(not(stage0))]
+    #[rustc_const_unstable(feature = "const_ptr_nonnull")]
```

Comment:
you can use `#[cfg_attr(not(stage0), rustc_const_unstable(feature = "const_ptr_nonnull"))]` in order to not need two items.

Reply:
Good point, will do it!